packages/generator-single-spa/src/react/templates/package.json

```diff
@@ -5,7 +5,7 @@
     "analyze": "webpack --mode=production --env.analyze=true",
     "lint": "eslint src --ext js<% if (typescript) { %>,ts,tsx<% } %>",
     "format": "prettier --write './**'",
-    "test": "jest",
+    "test": "cross-env BABEL_ENV=test jest",
     "watch-tests": "jest --watch",
     "coverage": "jest --coverage"
```

Comment:
the BABEL_ENV is needed everywhere jest is used or am I missing something?

Reply:
👍 good catch, yes it is needed when running the coverage command, too. I will add it.

Reply:
Updated